mautic/mautic
# New webhook: Contact Company Subscription Change

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
New webhook event send when contact's companies are changed (added/removed).

Response return 
- added (true or false in case if company is removed from contact)
- contact object
- company object in this event (added/removed)

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create webhook with **Contact Company Subscription Change Event** (test endpoint could be take from webhook.site)
![image](https://user-images.githubusercontent.com/462477/70643782-ea71cd00-1c41-11ea-9c77-30dc16dfb056.png)
3. Try create contact + assign/remove companies
4. Check webhook.site if your response contains
- added (true or false if removed)
- contact
- company 

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
